control

```diff
@@ -11,6 +11,7 @@ Build-Depends: debhelper-compat (= 13), meson (>= 0.53.2)
 Build-Depends-Arch: libbrotli-dev,
                     libcurl4-openssl-dev,
                     libevent-dev,
+                    libzstd-dev,
```

Comment:
@Tachi107, does he need it in the `Build-Depends-Arch` stanza as well?

Reply:
Sure. Adding both in `Build-Depends` and `Depends` means "libzstd-dev is needed both for building and consuming the libpistache package"

Reply:
No, sorry, I meant specifically `Build-Depends-Arch` as opposed to just `Build-Depends`.

Reply:
Build dependencies listed in the `Build-Depends-Arch` filed need to be satisfied when building architecture-dependent packages (i.e. packages marked with `Architecture: any`), while deps listed in `Build-Depends` need to also be present when building any kind of package, or when just running the `clean` target.

So `Build-Depends-Arch` is more correct here, since pistache doesn't build arch-independent packages and `libstd-dev` isn't needed to run the `clean` target.

This comes from [Debian Policy section 7.7](file:///usr/share/doc/debian-policy/policy.html/ch-relationships.html#relationships-between-source-and-binary-packages-build-depends-build-depends-indep-build-depends-arch-build-conflicts-build-conflicts-indep-build-conflicts-arch) (need to install the `debian-policy` package to open the link)